tests/console/yast2_dns_server.pm

```diff
@@ -25,10 +25,10 @@ sub assert_running() {
     my $running = shift;
 
     if ($running) {
-        assert_script_run 'systemctl is-active named | grep -E "^active"';
+        assert_script_run '(systemctl is-active named || true) | grep -E "^active"';
```

Comment:
Isn't assert_script_run "systemctl is-active named" enough? I think you don't need to grep for ^active string..
From documentation: is-active Returns an exit code 0 if at least one is active, or non-zero otherwise